tests/providers/gcp/gcp_test.py

```diff
@@ -722,7 +721,8 @@ def testGetMD5Object(self, mock_gcs_api):
     md5_base64 = FAKE_GCS.GetMD5Object('gs://Fake_Path')
     self.assertEqual('fakehash', md5_base64)
     md5_hex = FAKE_GCS.GetMD5Object('gs://Fake_Path', in_hex=True)
-    self.assertEqual(base64.b64decode('fakehash').hex(), md5_hex)
+    # '7da91e85ab21' = base64.b64decode('fakehash').hex()
+    self.assertEqual('7da91e85ab21', md5_hex)
```

Comment:
I guess what I meant here is that the string `fakehash` is not a great choice to use as a test value because you end up comparing `7da91e85ab21` to an md5 value, and this may be confusing to someone trying to figure out what md5_hex contains. Better use a value that makes semantic sense (a b64 encoding of an actual md5 hash)

Reply:
done